lib/monitor/monitor.go

```diff
@@ -18,31 +18,51 @@ import (
 var Sink = "0.0.0.0"
```

Comment:
I did not find a adequate name between what is sent from the client (a measure) and what the monitor actually measures/takes/stores/computes (a "measurement"). Please if you have any ideas...